infrastructure/scripts/test_gpt5_api_access.py

```
@@ -0,0 +1,309 @@
+    GPT5MigrationClient,
+    GPT5MigrationConfig,
+    GPTModel,
+)
+
+logger = structlog.get_logger(__name__)
+
+
+class GPT5APITester:
+    """Test GPT-5 API access and fallback mechanisms"""
+
+    def __init__(self):
```

Ruff (0.14.4)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)